src/components/UserInfo.vue

```diff
@@ -382,6 +411,16 @@ export default {
         })
         .catch(console.error);
     },
+    async toggleCookies(scope, status) {
```

Comment:
I think `toggleCookies` should switch status internally, otherwise, it should be called `setCookies`